client/plots/barchart.js

```diff
@@ -13,7 +13,7 @@ import { renderTable } from '../dom/table'
 import { fillTermWrapper } from '#termsetting'
 import { getColors } from '#shared/common'
 import { mclass } from '#shared/common'
-import { plotColor } from '../shared/common'
+import { isNumericTerm, plotColor } from '../shared/common'
```

Comment:
should always import from `#shared/common` also can combine line 14-16 into one import

Reply:
Done